### DISCUSSION TITLE
How to Represent Python Infinity in Simple Ways

### DISCUSSION THREAD

**Discussion by Optymizee** [magbanum/Python-Community-Discussions#44] [General]
To make it simple I have read a blog on how to make it simple if anyone can tell me if is it correct or just a made-up?

[https://optymize.io/blog/how-to-represent-python-infinity-in-simple-ways](https://optymize.io/blog/how-to-represent-python-infinity-in-simple-ways/)

I am about to start learning, and also please suggest to me where should i learn.

**Comment by vatche-t** [comment]
representing infinity in Python typically involves using the float type with the 'inf' string, which can be done as float('inf') for positive infinity or float('-inf') for negative infinity. This method is a standard and correct approach in Python programming for handling infinity in calculations or comparisons.

For learning Python, I recommend starting with resources like the official Python documentation, which is thorough and beginner-friendly. Websites like Coursera, edX, and Codecademy offer structured courses, some of which are free. Interactive platforms like LeetCode and HackerRank can help you practice coding problems. Finally, YouTube channels and Python-focused blogs provide valuable tutorials and tips for beginners.
